projects/example-app/src/app/app.config.ts

```diff
@@ -60,22 +54,27 @@ import { AppComponent } from '@example-app/core/containers';
      *
      * See: https://github.com/zalmoxisus/redux-devtools-extension
      */
-    StoreDevtoolsModule.instrument({
+    provideStoreDevtools({
       name: 'NgRx Book Store App',
       // In a production build you would want to disable the Store Devtools
       // logOnly: !isDevMode(),
     }),
 
     /**
-     * EffectsModule.forRoot() is imported once in the root module and
-     * sets up the effects class to be initialized immediately when the
-     * application starts.
+     * The provideEffects() function is used to register effect classes
+     * so they are initialized when the application starts.
      *
      * See: https://ngrx.io/guide/effects#registering-root-effects
      */
```

Comment:
I think we can remove the links as there's no distinction between root/feature effects.

```suggestion
     * The provideEffects() function is used to register effect classes
     * so they are initialized when the application starts.
     */
```

Reply:
Done